### PR TITLE
feat(web/applications): create applicant form pages(boas-148)

### PR DESCRIPTION
**Pull request by lutamKC**
## Describe your changes

- controllers and basic templates for every step in the applicant section of the form
- applicant info type page that saves in the session the types of applicant info that's going to be saved
- guard that prevents not allowed steps to be provided
- a service returning all the info types
- a service returning prev and next allowed page path

## Issue ticket number and link
https://pins-ds.atlassian.net/browse/BOAS-148

## Type of change 🧩

- [ ] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)
- [ ] Documentation update
- [ ] Other (please explain in the description section above)

## Checklist before requesting a review

- [ ] I have performed a self-review of my own code
- [ ] I have double checked this work does not include any hardcoded secrets or passwords
- [ ] I have made corresponding changes to the documentation
- [ ] I have provided details on how I have tested my code
- [ ] I have referenced the ticket number above
- [ ] I have provided a description of the ticket
- [ ] I have included unit tests to cover any testable code changes

### CODE REVIEW

File: apps/web/src/server/applications/applications.types.d.ts [hunk 1]
```diff
@@ -22,7 +29,7 @@ export interface Application {
 }
 
 export interface OptionsItem {
-	id: number;
+	id?: number;
```
Comment (francescobrenci-AR) [inline apps/web/src/server/applications/applications.types.d.ts:32]: wouldn't you expect this to be required?

**Review by francescobrenci-AR — APPROVED** [review]

File: apps/web/src/server/applications/applications.types.d.ts (hunk 1 above)
Comment (lutamKC) [reply apps/web/src/server/applications/applications.types.d.ts:32]: at the moment it's not required as the options for applicant info types don't really need an id.
i'm trying to keep the OptionsItem interface the most generic as possible and then below in the file I'm being more specific with types for ZoomLevel, Region and Sector

**Pull request merged**